### PR TITLE
data: donaina

### PR DESCRIPTION
**Pull request by donaina**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5255"></a>

### CODE REVIEW

**Review by Sindhuinti — DISMISSED** [review]
Looks good 👍🏻

File: data/donaina.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"name": "Aina Ayoola",
+	"displayStatsPublic": "true",
```
Comment (priyankarpal) [inline data/donaina.json:3]: hi, @donaina after the recent update "displayStatsPublic": "true",` is no longer needed. You can delete this line. 
``` suggestion
	
```

File: data/donaina.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	"type": "personal",
+	"bio": "I am a QA engineer & Database Administrator with experience that span across Infrastructure,Networking, Application support & QA",
+	"avatar": "https://github.com/donaina.png",
```
Comment (priyankarpal) [inline data/donaina.json:6]: Also, you can delete the `avatar` line.
```suggestion

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by iamthecloverly — APPROVED** [review]

**Review by Pradumnasaraf — COMMENTED** [review]
Looks good to merge. Welcome to LinkFree 🎉

**Pull request merged**